components/apps_form/apps_form_field/apps_form_field.tsx

```diff
@@ -87,6 +92,10 @@ export default class AppsFormField extends React.PureComponent<Props> {
         this.providers = providers;
     }
 
+    onClear = () => {
+        this.handleSelected(null);
```

Comment:
At a different point, we were considering the empty list for the default values. Should we consider the empty list here too?

Reply:
Thank you @larkox, I've moved where that's done to prepare for supporting multiselect users and channels in the future